unit_tests/log_transfer/test_main_chkpt_info.cpp

```diff
@@ -0,0 +1,800 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
+
+#include "client_credentials.hpp"
+#define private public
+#include "log_checkpoint_info.hpp"
+#undef private
+#include "log_impl.h"
+#include "log_lsa.hpp"
+#include "log_record.hpp"
+#include "mem_block.hpp"
+#include "thread_entry.hpp"
+#include "system_parameter.h"
+
+#include <algorithm>
+#include <cstdlib>
+#include <map>
+#include <random>
+#include <vector>
+
+using namespace cublog;
+
+class test_env_chkpt
+{
+  public:
+    test_env_chkpt ();
+    test_env_chkpt (int size_trans, int size_sysops);
```

Comment:
For memsize & counts, favor size_t type.
```suggestion
    test_env_chkpt (size_t size_trans, size_t size_sysops);
```